### ISSUE TITLE
Webstorm incompatibility

### ISSUE THREAD

**Issue by wayofthefuture** [webpack/webpack-dev-server#845]
I've been programming with Webstorm for a very long time, and it is one of the best and very popular IDE's for javascript. But for some reason, webpack watch with dev server doesn't work if the file is saved to a temporary file then renamed back to the original file.

See: File, Settings, Appearance & Behavior, System Settings, "Use safe write"

Webstorm uses safe write, which saves the changes to a temporary file first. I don't think it would a problem with webstorm because this is the intended behavior. It took me a while to find out which setting was making dev-server not work properly. Just thought I'd let you know in case you think it should be resolved.

Basically, webpack detects changes and rebuilds for use with dev-server, but the browser never sees the file changes. (Windows 10)

**Comment by SpaceK33z** [comment]
Thanks, but this is a known issue and [also documented](https://webpack.js.org/guides/development/#adjusting-your-text-editor). Other text editors also have this, but so far there hasn't been any workaround.
